### PR TITLE
Add name field to workshop table

### PR DESCRIPTION
**Pull request by TurnerRiley**
Adds `name` field to the `pd_workshops` table.

## Links
Jira ticket: [here](https://codedotorg.atlassian.net/jira/software/c/projects/ACQ/boards/64?assignee=60d62161f65054006980bd71&selectedIssue=ACQ-2835)
Tech plan: [here](https://docs.google.com/document/d/1R0X9LhfxA2b1BpSGryXaPekyhm2lkf3WeuT9WkWE8nw/edit?tab=t.0)

## Testing story
Successfully ran migration.

### CODE REVIEW

**Review by drizco — APPROVED** [review]
lgtm! though I'm not sure how you resolve a conflict in the schema.rb file 🤔 

you may need to rollback your migration, rebase with staging-next, and then run it again? 🤷

**Comment by bethanyaconnor** [comment]
In this case, I think it's okay to just accept your version of the schema.rb conflict. It actually doesn't matter that much because I believe the staging machine regenerates this file, but it needs to be committed for CI to run (I think?)

**Review by bethanyaconnor — APPROVED** [review]

File: dashboard/db/schema.rb [hunk 1]
```diff
@@ -10,7 +10,11 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
+<<<<<<< HEAD
+ActiveRecord::Schema.define(version: 2024_12_04_175716) do
+=======
 ActiveRecord::Schema.define(version: 2024_12_03_163649) do
+>>>>>>> staging-next
```
Comment (bethanyaconnor) [inline dashboard/db/schema.rb]: Sorry, I might not have been clear ... you should resolve the conflict but I think accepting your change here is fine.

Comment (TurnerRiley) [reply dashboard/db/schema.rb]: Oh whoops that's odd? I did resolve it and clicked "Accept" in the VSCode display, but then it kept the `<<<< HEAD` lines? I'll fix that!

**Pull request closed without merging**